packages/suite/src/views/wallet/tokens/TokensNavigation.tsx

```diff
@@ -133,25 +60,33 @@ export const TokensLayoutNavigation = ({
     }, [account.symbol, account.index, account.accountType, setSearchQuery]);
 
     return (
-        <Wrapper>
-            <List>
-                <Item
-                    route="wallet-tokens-coins"
-                    title="TR_COINS"
+        <Row alignItems="center" justifyContent="space-between" margin={{ bottom: spacings.md }}>
```

Comment:
Why use two Rows? I think they can be combined.

Reply:
No, in the end both rows are needed